gcc/config/aarch64/aarch64-coff.h

```diff
@@ -63,6 +63,16 @@
   assemble_name ((FILE), (NAME)),		\
   fprintf ((FILE), ",%u\n", (int)(ROUNDED)))
 
+#define ASM_OUTPUT_ALIGNED_LOCAL(FILE, NAME, SIZE, ALIGNMENT)  \
```

Comment:
The other usage of `ASM_OUTPUT_ALIGNED_LOCAL` here [https://github.com/Windows-on-ARM-Experiments/gcc-woarm64/blob/woarm64/gcc/varasm.cc#L2783](https://github.com/Windows-on-ARM-Experiments/gcc-woarm64/blob/woarm64/gcc/varasm.cc#L2783) seems very similar to the new implementation. What is the main difference between the fallback implementaiton and the new one? Wound't be better just to re-define `BIGGEST_ALIGNMENT` instead of the full coputation?

Reply:
Not sure that AArch64 maintainers will be happy about BIGGEST_ALIGNMENT change.

Reply:
Also I suspect it might be even bigger than 32 which is used in FFmpeg.

Reply:
> Not sure that AArch64 maintainers will be happy about BIGGEST_ALIGNMENT change.

What is the difference between overwriting just the `BIGGEST_ALIGNMENT` in aarch64-coff.h compared to overwriting the whole `ASM_OUTPUT_ALIGNED_LOCAL` in terms of acceptance?

Reply:
I think this what will figure out during path review with the community. For now I prefer not to change it.